src/System.Private.CoreLib/shared/System/InsufficientMemoryException.cs

```diff
@@ -26,8 +25,10 @@ namespace System
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public sealed class InsufficientMemoryException : OutOfMemoryException
     {
+        // There may be a problem here interacting with the ResourceManager in out of memory conditions,
```

Comment:
Where is this fallback? I am not sure where this comment came from.

Reply:
It came from corert.